app/assets/stylesheets/article-show.scss

```diff
@@ -352,6 +352,7 @@ article {
         color: $black;
         padding: calc(0.3vw + 14px);
         background: darken($lightest-gray, 12%);
+        background: var(--theme-series-article-background, darken($lightest-gray, 12%));
```

Comment:
You could use the custom `themeable` mixin to avoid repeating the property twice: 

https://github.com/thepracticaldev/dev.to/blob/180935bd37c29f7d51cafe079b843957a036119b/app/assets/stylesheets/_mixins.scss#L9-L12

Reply:
This should be refactored now 😃